config.json

```diff
@@ -53,674 +53,441 @@
         "slug": "hello-world",
         "name": "Hello World",
         "uuid": "3eeadff6-82b6-43be-8834-d9ef258e454d",
-        "practices": [],
-        "prerequisites": [],
-        "difficulty": 1,
-        "topics": [
-          "strings"
-        ]
+        "practices": ["string","input and output"],
+        "prerequisites": ["baics"],
```

Comment:
Found a typo here i think you mean basic or?

Reply:
Yep thanks..will update it by tomorrow